### DISCUSSION TITLE
More Languages , How and When ?

### DISCUSSION THREAD

**Discussion by fire17** [coqui-ai/TTS#1161] [General]
Hi there 👋
Wondering how or when it will be possible to get more languages available into the model. For me Hebrew is the common language and would like to process or output voice in my language .

Thanks a lot and have a good one! :)

**Comment by erogol** [comment]
We don't have an immediate plan to train a Hebrew model. 

If you know a public dataset, you can point to it here and someone from the 🐸 community might like to train a model. 

(I move this to the discussions as it is out of the Feature Request definition. )

**Comment by thewh1teagle** [comment]
I would like to add hebrew support too. 
Is there some tutorial or showcase of someone who added new language support for that?
I don't mind to create myself the entire dataset for that and do whatever needed for that.

**Comment by fire17** [comment]
You guys say ~1100 languages...
Where's the list? Couldn't find it
Thanks and all the best!

**Reply by fire17** [reply]
@erogol ? 🙏
Also any updates on hebrew tts ? Thx and all the best! :)

**Reply by erogol** [reply]
https://tts.readthedocs.io/en/latest/inference.html#example-text-to-speech-using-fairseq-models-in-1100-languages

**Comment by isolveit-aps** [comment]
There is a very good quality "BLARK" for Faroese available under BY-CC 4.0. Can someone tell me if I can easily start experimenting with training a Faroese TTS model in Faroese language? Or how would this normally get to reality in Coqui?

**Comment by Lazarus-Leonardo** [comment]
You do not speak Hebrew, you speak Yiddish. Nobody speaks Hebrew anymore.
